src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraPreviewControlRenderer.cs

```diff
@@ -268,5 +267,32 @@ private void PrepareAndStartCamera()
                 camera.SetParameters(cameraParameters);
             }
         }
+
+        private IList<CameraFocusMode> RetrieveSupportedFocusModes()
```

Comment:
Why return IList - doesn't this mean that you can add/remove to the list. Probably should just be IEnumerable?

Reply:
was a starter to return an interface, will look to change to IReadOnlyList

Reply:
yep that sounds good